arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -1854,6 +1937,58 @@ static void esp_evt_work_cb(FAR void *arg)
     }
 }
 
+/****************************************************************************
+ * Name: wifi_env_is_chip
+ *
+ * Description:
+ *   Config chip environment
+ *
+ * Returned Value:
+ *   True if on FPGA or false if not.
+ *
+ ****************************************************************************/
+
+static bool wifi_env_is_chip(void)
+{
+  return true;
+}
```

Comment:
Does this still need to be here?

Reply:
This function's description is not right, but this function is also needed, because it one of wifi adapter structure member.